### PR TITLE
feat(script.js, style.css): added toggle buttons with buttons for pairing filiters

### LINKED ISSUE #33: [Feat]: Intern Pairing Toggle
### Is your feature request related to a problem? Please describe.

Currently the application does not have a way to filter pairing by location and department only.

### Describe the solution you'd like.

Need to add toggle buttons for department and location based pairing. Preferably a drop down menu that has both those options. The functionality will check if location is === location or the same with department. This functionality will be added to generate random pair function.

### Describe alternatives you've considered

no response

### Additional context

no response

### PR DESCRIPTION
**Pull request by vlomeli**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Changes
_List Changes Introduced by this PR_
1. made changes to html and script to add css and js to make it responsive.

## Purpose
need buttons to use filters.

## Approach
I had to use chat gpt to make the buttons pop up the way that they do with a set timeout. 

## Pre-Testing TODOs
none

## Testing Steps
click the pairing button and hover over the other buttons that pop up.

## Learning
learned one way to use set timeout.

_Links to blog posts, patterns, libraries or addons used to solve this problem_

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #33

### CODE REVIEW

File: script.js
Comment (karllegson) [inline script.js:1]: Looks good, it's prettier

**Review by karllegson — COMMENTED** [review]
Everything is good

**Review by joavelar — APPROVED** [review]
Good to go.

**Review by karllegson — APPROVED** [review]
Goodjob

**Pull request merged**